src/main/java/hudson/plugins/cppncss/CppNCSSGhostwriter.java

```diff
@@ -46,7 +46,8 @@ public CppNCSSGhostwriter(String reportFilenamePattern, Integer functionCcnViola
         this.targets = targets;
     }
 
-    public boolean performFromMaster(AbstractBuild<?, ?> build, FilePath executionRoot, BuildListener listener)
+    @Override
+    public boolean performFromMaster(Run<?, ?> build, FilePath executionRoot, TaskListener listener)

```

Comment:
🐜  Binary incompatible method signature, but I *doubt* there are plugins depending on this API